### PR TITLE
Jarrod Review

### PR DESCRIPTION
**Pull request by jarrodruby**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -13,27 +16,34 @@ router.get("/", (_, res) => {
...
 		});
 		const payload = ticket.getPayload();
-		res.status(200).json({ message: "success" });
+		const { name, email } = payload;
+		let user = await Users.findOne({ where: { email } });
+		if (!user) {
+			persistNewUser(name, email, role, token);
+		} else {
+			res.status(200).json({ message: "success" });
+		}
```
Comment (jarrodruby) [inline server/api.js:46]: Created a bug for what we are doing here:  https://github.com/users/karleenmsrichards/projects/4/views/1?pane=issue&itemId=35131052

Comment (jarrodruby) [reply server/api.js:46]: We always want to return a reponse code

```javascript
let user = await Users.findOne({ where: { email } });
     try {
		if (!user) {
			persistNewUser(name, email, role, token);
		} 
			res.status(200).json({ message: "success" });
		}
} catch(e){
             res.status(500).json({ message: error });
}

```

File: server/sequelize/migrations/20230804001548-create-tokens-table.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      token: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
```
Comment (jarrodruby) [inline server/sequelize/migrations/20230804001548-create-tokens-table.js:15]: A User can have many tokens as when the token expires Google will generate a new one

File: server/api.js (hunk 1 above)
Comment (farzaneh-haghani) [reply server/api.js:48]: I will add return with my next PR.

File: client/src/App.js [hunk 3]
```diff
@@ -4,17 +4,25 @@ import ContactUs from "./pages/ContactUs";
...
-	</div>
-);
+const App = () => {
+	const { isLoggedIn } = useAuth();
+	return (
+		<div>
+			<Header />
+			<Routes>
+				<Route path="/" element={<Home />} />
+				{isLoggedIn && <Route path="/sign-up" element={<SignUpForm />} />}
```
Comment (jarrodruby) [inline client/src/App.js:18]: SHouldn't this be when they are not logged in?

`{!isLoggedIn && <Route path="/sign-up" element={<SignUpForm />} />}`

Comment (farzaneh-haghani) [reply client/src/App.js:18]: It's not signUp to use the app, It's sign up for become a provider.
It's for people try to access this page from url.

Comment (farzaneh-haghani) [reply client/src/App.js:18]: At the moment, we have some bugs. Nearly end of next week, we should go over them.

**Pull request merged**